src/nyc_trees/apps/survey/urls/blockface.py

```diff
@@ -41,11 +41,6 @@
     url(r'^checkout-confirmation/$', blockface_reservations_confirmation_page,
         name='blockface_reservations_confirmation_page'),
 
-    # Note: this must be kept in sync with the hardcoded url in
-    # js/src/mapUtil.js
-    url(r'^(?P<blockface_id>\d+)/$', blockface,
-        name='blockface'),
```

Comment:
Removing this view has broken the reservations page.  I think we need to add it back in.